test/server/cards/08-MotC/TheSpearRushesForth.spec.js

```diff
@@ -0,0 +1,104 @@
+                    attackers: [this.brashSamurai, this.dojiWhisperer],
+                    defenders: [this.dojiChallenger]
+                });
+                this.player2.pass();
+                this.player1.clickCard(this.theSpearRushesForth);
+                expect(this.player1).toHavePrompt('Conflict Action Window');
+            });
+
+            it('should work if the player controls a participating honored character', function() {
+                this.brashSamurai.honor();
+                this.initiateConflict({
+                    attackers: [this.kakitaKaezin, this.brashSamurai, this.dojiWhisperer],
+                    defenders: [this.dojiChallenger]
+                });
+                this.player2.pass();
+                this.player1.clickCard(this.theSpearRushesForth);
+                expect(this.player1).toHavePrompt('The Spear Rushes Forth');
+                expect(this.player1).toHavePrompt('Choose a character');
+            });
+
+            it('should discard an honored status token as a cost', function() {
+                this.brashSamurai.honor();
+                this.dojiChallenger.honor();
+                this.initiateConflict({
+                    attackers: [this.kakitaKaezin, this.brashSamurai, this.dojiWhisperer],
+                    defenders: [this.dojiChallenger]
+                });
+                this.player2.pass();
+                this.player1.clickCard(this.theSpearRushesForth);
+                expect(this.player1).toHavePrompt('Choose a character');
+                this.player1.clickPrompt('Pay Costs First');
+                expect(this.player1).toBeAbleToSelect(this.kakitaKaezin);
+                expect(this.player1).toBeAbleToSelect(this.brashSamurai);
+                expect(this.player1).not.toBeAbleToSelect(this.dojiChallenger);
+                expect(this.player1).not.toBeAbleToSelect(this.dojiWhisperer);
+                this.player1.clickCard(this.kakitaKaezin);
+                expect(this.kakitaKaezin.isHonored).toBe(false);
+            });
+
+            it('should bow targeted character', function() {
+                this.brashSamurai.honor();
+                this.initiateConflict({
+                    attackers: [this.kakitaKaezin, this.brashSamurai, this.dojiWhisperer],
+                    defenders: [this.dojiChallenger, this.kakitaYoshi]
+                });
+                this.player2.pass();
+                this.player1.clickCard(this.theSpearRushesForth);
+                expect(this.player1).toBeAbleToSelect(this.dojiChallenger);
+                expect(this.player1).toBeAbleToSelect(this.kakitaYoshi);
+                this.player1.clickCard(this.kakitaYoshi);
+                this.player1.clickCard(this.kakitaKaezin);
+                expect(this.kakitaYoshi.bowed).toBe(true);
+                expect(this.kakitaKaezin.isHonored).toBe(false);
+            });
+
+            it('should work even if steward of law is present in the conflict', function() {
```

Comment:
Steward of law wouldn't prevent going from honored to ordinary anyway, a better test would be with Young Harrier which would normally prevent dishonoring (but not the removal of an honor status token).

Reply:
I added the Young Harrier test and it indeed prevents the cost from being paid in this case.